### PR TITLE
make market a component

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/market/components/Carousel.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Carousel } from 'antd';
+
```
Comment (enyeInc) [inline src/components/market/components/Carousel.js:3]: remove unneeded space

File: src/components/market/components/Market.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    };
+
+    handleCancel = () => {
+        this.setState({ visible: false });
+    };
+
+    render() {
+        const { visible } = this.state;
+        const { cost, image, name } = this.props.data;
+        const description = tag => (
```
Comment (enyeInc) [inline src/components/market/components/Market.js]: this doesn't need to be a function

File: src/components/market/components/MarketList.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { List } from 'antd';
+import PropTypes from 'prop-types';
+
+import Market from './Market';
+import CarouselImages from './Carousel';
+
```
Comment (enyeInc) [inline src/components/market/components/MarketList.js]: remove unneeded space

File: src/components/market/constants.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        EMAIL: 'Email:',
+        LOCATION: 'Location:',
+        PHONE: 'Phone #:',
+        PRODUCERS_NAME: `Producer's Name:`,
+        QUANTITY: 'Available Quantity:',
+    },
+    TAG_COLOR: 'gold',
+};
+
+export const EXAMPLE_PRODUCTS = [
```
Comment (enyeInc) [inline src/components/market/constants.js]: get the products from the backend not from this constants file

File: src/components/market/components/Carousel.js (hunk 1 above)
Comment (joshuaobinna) [reply src/components/market/components/Carousel.js:3]: ok

File: src/components/market/components/Market.js (hunk 2 above)
Comment (joshuaobinna) [reply src/components/market/components/Market.js]: ok

File: src/components/market/components/MarketList.js (hunk 3 above)
Comment (joshuaobinna) [reply src/components/market/components/MarketList.js]: ok

File: src/components/market/constants.js (hunk 4 above)
Comment (joshuaobinna) [reply src/components/market/constants.js]: ok

**Review by enyeInc — COMMENTED** [review]
Looks good to me

**Pull request merged**